circuit-foundation/src/commonMain/kotlin/com/slack/circuit/foundation/CircuitContent.kt

```diff
@@ -53,6 +53,10 @@ public fun CircuitContent(
           onNavEvent(NavEvent.Pop)
           return null
         }
+
+        override fun peek(): Screen? {
+          return null
```

Comment:
This should just return the screen we're on, no?

Reply:
Yes, right. I changed in 3c3bd5ae6d9ca54d0bd72e81806da82bb9d0a36f 👍